lib/pod/command/dependencies.rb

```diff
@@ -32,6 +34,16 @@ def initialize(argv)
         @produce_graphviz_output = argv.flag?('graphviz', false)
         @produce_image_output = argv.flag?('image', false)
         @use_podfile_targets = argv.flag?('use-podfile-targets', false)
+        @ranksep = "0.75"
+        @nodesep = "0.25"
+        ranksep = argv.all_options('ranksep').at(0)
```

Comment:
can this use `argv.option('rankest', '0.25')` ?

Reply:
Yep, it's briefer